src/components/layouts/Groundfloor.vue

```diff
@@ -61,5 +72,9 @@ export default defineComponent({
 
         transition: opacity var(--transition-time) var(--nimiq-ease);
     }
+
+    .settings {
```

Comment:
Maybe this should be better in App.vue, as I see it, there are two problems:
- `.settings` selector is too general and the box in the sidebar with the `settings` class is also getting this class
- The rest of elements have their dimensions in App.vue, not Groundfloor.